Sasquatch/Sasquatch/ViewControllers/MSDocumentDetailsViewController.swift

```diff
@@ -72,6 +74,31 @@ class MSDocumentDetailsViewController: UIViewController, UITableViewDelegate, UI
     self.presentingViewController?.dismiss(animated: true, completion: nil)
   }
 
+  @IBAction func refreshButtonClicked(_ sender: Any) {
+    var partition = documentContent?.partition ?? ""
+    if (partition.contains("user")) {
+        partition = "user"
+    }
+    self.appCenter.readDocumentWithPartition(partition, documentId: documentContent?.documentId ?? "", documentType: MSDictionaryDocument.self, completionHandler: { (document) in
+        self.documentContent = document
+        DispatchQueue.main.async {
+            self.tableView.reloadData()
+        }
+        
+        if (partition == "user") {
```

Comment:
We can probably factorize the code a bit here. The only difference is the array of documents